### ISSUE TITLE
Can clang compile 64-bit code?

### ISSUE THREAD

**Issue by wjk** [Microsoft/WinObjC#122]
I was trying to manually compile some code as 64-bit, but when I pass the `-m64` flag to WinObjC clang, it fails with the following error:

    failed to load c2 (file not found)

It seems that the MSVC 64-bit backend is missing. However, the rest of my project is also marked as 64-bit (using the MSBuild `$(Platform)` variable), and compiles just fine. What is going on here? Is there another flag I need to pass to Clang to get it to compile correctly as 64-bit?

**Comment by wjk** [comment]
As it turns out, _none_ of my code is compiled as 64-bit. Even though the MSBuild `$(Platform)` variable was set to `x64`, inspecting the resulting object files using `dumpbin.exe` reveals that they are compiled as `x86`. It seems that none of the compiler toolchain supports 64-bit executables. @s5msft @jmsaunders-ms is there any way to change this?

**Comment by jmsaunders-ms** [comment]
You're right, clang/c2 does not presently support codegen for x64. This is something we're aware of and it is being planned.

ARM support and optimizations have higher priority than x64 code generation, so I would expect to see support after those.

**Comment by DHowett-MSFT** [comment]
Clang can now do this, but the rest of WinObjC can't!
